code/src/components/eachTask/EachTask.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import React from 'react';
+import { useDispatch } from 'react-redux';
+import { tasks } from 'reducers/tasks';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faSquare, faSquareCheck, faTrashCan } from '@fortawesome/free-regular-svg-icons';
+import { ButtonStyles } from 'components/buttons/Buttons.styles';
+import { Timestamp } from './Timestamp';
+import { DeleteTaskStyles, EachTaskStyles, TaskContentStyles, ToggleButtonStyles } from './EachTask.styles';
+
+export const EachTask = ({ singleTask }) => {
+  const dispatch = useDispatch();
+  return (
+    <EachTaskStyles>
+      <ToggleButtonStyles>
+        {!singleTask.complete && (<ButtonStyles type="button" onClick={() => dispatch(tasks.actions.toggleComplete(singleTask))} buttonWidth="30px" buttonText="black" buttonColor="white" buttonBorder="none"><FontAwesomeIcon icon={faSquare} /></ButtonStyles>)}
+        {singleTask.complete && (<ButtonStyles type="button" onClick={() => dispatch(tasks.actions.toggleComplete(singleTask))} buttonWidth="30px" buttonText="black" buttonColor="white" buttonBorder="none"><FontAwesomeIcon icon={faSquareCheck} /></ButtonStyles>)}
```

Comment:
interesting that you used checkbox icons instead of `input type = ‘checkbox’`. Is there a reason for this?

Reply:
I knew that I could make a button do what I wanted it to do quite easily. So, I decided to go with that so I could focus on learning the other, newer aspects of the project.